src/xcode/ENA/ENA/Source/RootCoordinator.swift

```diff
@@ -117,7 +117,8 @@ class RootCoordinator: RequiresAppDependencies {
 		tabBarController.tabBar.tintColor = .enaColor(for: .tint)
 		tabBarController.tabBar.barTintColor = .enaColor(for: .background)
 		tabBarController.setViewControllers([homeCoordinator.rootViewController, checkInCoordinator.viewController, diaryCoordinator.viewController], animated: false)
-
+		
```

Comment:
Forget the old white space. This is NEW white space!
Are we even supposed to call it 'white' space anymore? How about empty space?